File: 백준/Bronze/2675. 문자열 반복/문자열 반복.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        StringBuilder sb = new StringBuilder();
        StringTokenizer st;
        int tc = Integer.parseInt(br.readLine());
        for(int i = 0; i < tc; i++){
            st = new StringTokenizer(br.readLine());
            int iter = Integer.parseInt(st.nextToken());
            String str = st.nextToken();
            for (int j = 0; j < str.length(); j++){
                String chr = String.valueOf(str.charAt(j));
                for (int k = 0; k < iter; k++){
                    sb.append(chr);
                }
            }
            sb.append("\n");
        }
        System.out.println(sb);
        br.close();
        bw.flush();
        bw.close();
    }
}

File: 백준/Bronze/2675. 문자열 반복/문자열 반복.kt
import java.io.*
import java.util.*

fun main() = with(BufferedReader(InputStreamReader(System.`in`))) {

    val t = readLine().toInt()
    val sb = StringBuilder()

    repeat(t) {
        val st = StringTokenizer(readLine())

        val r = st.nextToken().toInt()
        val s = st.nextToken()

        for(element in s) {

            repeat(r) {
                sb.append(element)
            }
        }
        sb.append("\n")
    }

    println(sb)
}